source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -1,79 +1,77 @@
+    <?endif;?>
+    <hr>
+      <li class='caMenuItem debugging'><?tr("Debugging");?></li>
+    </ul>
   </div>
-</span>
\ No newline at end of file
+  </span>
+  <div class='mainArea'>
+  <div class='spinnerBackground'></div>
+  <div class='spinner fixed'></div>
+  <strong><font size='3'></strong></font>
```

HTMLHint (1.5.0)

[error] The value of attribute [ size ] must be in double quotes.

(attr-value-double-quotes)